python/test/t_Event_system.py

```diff
@@ -135,15 +135,11 @@ def subset_event(ev):
 
 def f1py(x):
     x0 = ot.Sample(x).getMarginal(0)
-    print("eval f1")
-    assert x0.getSize() == n, "no batch eval"
     return x0
 
 
 def f2py(x):
     x1 = ot.Sample(x).getMarginal(1)
-    print("eval f2")
-    assert x1.getSize() > 1, "no batch eval"
```

Comment:
why remove these ? this tests for batch evaluation

Reply:
also you may include the test you proposed in the PR description

Reply:
> why remove these ? this tests for batch evaluation

Since the PR does not include a "getFrozenSample" method, these asserts fail because getFrozenRealization gets called.

Reply:
@josephmure : Can this suggestion be taken into account in your framework?

Reply:
Done in the latest commit